boefjes/plugins/kat_nessus/normalizer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id": "kat_nessus_normalize",
+    "consumes": [
+        "boefje_nessus"
```

Comment:
"boefje/nessus" would work. as the syntax that our runner uses. (it takes the name of the boefje, and prepends it with "boefje/"